### PR TITLE
Created database and table, added records in table, updated README file

### PR DESCRIPTION
**Pull request by Pazzo97**
### In this PR I did the following:

- Created a database named vet_clinic.
- Created a table named animals with the following columns:
    id: integer
    name: string
    date_of_birth: date
    escape_attempts: integer
    neutered: boolean
    weight_kg: decimal
- Copied the SQL I used in the previous point into the schema.sql file in the generated repository.
- Inserted the data into the animals' table

- Copied the SQL I used in the previous point into the data.sql file in the generated repository.
Wrote queries for the following quests:
   Find all animals whose name ends in "mon".
   List the name of all animals born between 2016 and 2019.
   List the name of all animals that are neutered and have less than 3 escape attempts.
   List the date of birth of all animals named either "Agumon" or "Pikachu".
   List name and escape attempts of animals that weigh more than 10.5kg
   Find all animals that are neutered.
   Find all animals not named Gabumon.
   Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 
   10.4kg or 17.3kg)

- Took screenshots of the results of your queries.
- Copied the SQL you used in the previous point into the queries.sql file in the generated repository.

### Screenshots of the queries result:

-  All animals whose name ends in "mon".
mon

-  List the name of all animals born between 2016 and 2019.
2016-2019-animals

- List the name of all animals that are neutered and have less than 3 escape attempts.
neutered-escape 3

- List the date of birth of all animals named either "Agumon" or "Pikachu".
Agumon-pikachu

- List name and escape attempts of animals that weigh more than 10.5kg
escape 10 5kg

- Find all animals that are neutered.
ever-neutered

- Find all animals not named Gabumon.
not-gabumon

- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 
   10.4kg or 17.3kg)
btn-10 4-17 3kg

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals(
+  id      		  	INT,
+  name    			  VARCHAR(250),
+  date_of_birth   DATE,
+  escape_attempts INT,
+  neutered     		BIT,
```
Comment (Kingstalux) [inline schema.sql]: - [ ] Good job so far 👍 Kindly make the `neutered` column `boolean` and not `bit` so as to follow the project requirements 😄

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals(
+  id      		  	INT,
```
Comment (Kingstalux) [inline schema.sql]: - [ ] (OPTIONAL)You can set the `id` as the primary key and also set it to not be null that is :
```node
 id int Primary key NOT NULL IDENTITY(1,1)
```

File: data.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (1, 'Agumon', '2020-02-03', 0, 1, 10.23);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (2, 'Gabumon', '2018-11-15', 2, '1', 8);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (3, 'Pikachu', '2021-01-07', 1, '0', 15.04);
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kgs) VALUES (4, 'Devimon', '2017-05-12', 5, '1', 11);
```
Comment (Kingstalux) [inline data.sql]: - [ ] Since `neutered` is a boolean, its value should be either true or false. Kindly rectify this here 👍

File: queries.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-12-31' AND '2019-01-01';
```
Comment (Kingstalux) [inline queries.sql]: - [ ] The range here should be from January 1st 2016 to December 31st 2019. Kindly correct that 😄

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Pazzo97, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
175083599-879bf673-fb93-406f-939c-de7e586bf6d9

## Highlights 👏 
✅ Good PR description 🔥 
✅ Professional REAMDE file 👍 
✅ Good job writing the queries 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Pazzo97,

Thank you for implementing changes requested by previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
appr3

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**